lib/shore/customers/client.rb

```diff
@@ -25,32 +26,24 @@ def initialize(options = {})
 
       def initialize_conn(options)
         base_url = options[:base_uri] || ENV['CUSTOMER_STORE_BASE_URI']
-        fail "Error: #{self.class} must be initialized with a \
-:base_url option or env variable CUSTOMER_STORE_BASE_URI \
-must be set." if base_url.blank?
+        fail initialize_failure if base_url.blank?
         secret = options[:secret] || ENV['CUSTOMER_STORE_SECRET']
 
         @conn = Faraday.new(url: base_url) do |faraday|
           # TODO@am: Configure Faraday here
           # faraday.request  :url_encoded
           # faraday.response :logger
+          faraday.response :json, content_type: 'application/json'
```

Comment:
`faraday_middleware` can parse the response for you based on the `content_type`, and it's a middleware so it will process the response before delivering it, and there's no need to call a `format_response` method.